packages/devtools_shared/test/helpers/extension_test_manager.dart

```diff
@@ -394,6 +394,10 @@ class TestPackageWithExtension {
   final bool requiresConnection;
   final bool isPubliclyHosted;
   final String? packageVersion;
+
+  /// The relative path from the extensions.
+  ///
+  /// Uses the paths separator for the current platform.
   final String relativePathFromExtensions;
```

Comment:
This was used in two places, one where a path was expected and one to write into Pubspec. Both of them had different wants for slashes, so I decided that because this is called `relativePath` it is best to be the native path separator, and in the Pubspec generation (below), normalize it to forward slashes.